osgp/protocol-adapter-dlms/osgp-protocol-adapter-dlms/src/main/java/org/opensmartgridplatform/adapter/protocol/dlms/domain/commands/utils/DlmsHelper.java

```diff
@@ -346,7 +349,9 @@ public CosemDateTimeDto convertDataObjectToDateTime(final DataObject object) thr
             final CosemDateTime cosemDateTime = object.getValue();
             dateTime = this.fromDateTimeValue(cosemDateTime.encode());
         } else {
-            this.logAndThrowExceptionForUnexpectedResultData(object, "ByteArray or CosemDateFormat");
+            //this.logAndThrowExceptionForUnexpectedResultData(object, "ByteArray or CosemDateFormat");
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') This block of commented-out lines of code should be removed. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://ci.opensmartgridplatform.org/sonarqube/coding_rules#rule_key=squid%3ACommentedOutCodeLine)